### PR TITLE
feat(dockerfiles): add support to docker secrets for configs (#58)

### LINKED ISSUE #58: Support more ways to control docker images other than environment variables
currently you control docker deployments with setting environment variables which is an easy solution but considered not secure. These environment variables contain connection strings and maybe other sensitive information. We should allow configuration via files and any other way that makes sense for running dynamico in a container.

### PR DESCRIPTION
**Pull request by mikeChatsky**
* Add support to pass sensitive config fields through docker secrets in addition to env variables (for security reasons).
* Configurable docker secrets fields: (s3-registry) access-key-id and secret-access-key.
* Add SECRETS_FILE_PATH env variable to customize secrets location inside the container.

fixes #58

### CODE REVIEW

**Comment by eladav** [comment]
Thanks for the contribution! I'll look at it as soon as I can

File: dockerfiles/s3-registry/index.ts [hunk 1]
```diff
@@ -4,17 +4,27 @@ import dynamico from '@dynamico/express-middleware';
 import { S3Storage } from '@dynamico/s3-storage';
 const { S3 } = require('aws-sdk');
 import cors from 'cors';
+import nconf from 'nconf';
```
Comment (eladav) [inline dockerfiles/s3-registry/index.ts]: Adding `nconf` is a good idea. Can you change other usages or `process.env.whatever` to `nconf.get`?

**Comment by eladav** [comment]
Generally this is a really good approach, and a good implementation (👍 for adding documentation 😃)  Can you add an implementation for the other implementations (azure blobs, redis) please?

File: dockerfiles/s3-registry/index.ts (hunk 1 above)
Comment (mikeChatsky) [reply dockerfiles/s3-registry/index.ts]: Sure, no problem 👍

**Comment by mikeChatsky** [comment]
@eladav Thank you for the feedback 😀
> Can you add an implementation for the other implementations (azure blobs, redis) please?
Since all the configs in the other implementations are already read from a file, using docker secrets through file is already possible. 
However, I agree we still can offer a clearer way to specify docker secrets (documentation and examples). I'll think of a way to implement this as soon as I can.

**Comment by eladav** [comment]
Since we have some issues with GH actions I'll merge it using admin super powers... we'll work on fixing this ASAP

**Pull request merged**